beta-src/src/webDiplomacyTheme.ts

```diff
@@ -95,6 +95,30 @@ const webDiplomacyTheme = createTheme({
         },
       },
     },
+    MuiSvgIcon: {
+      styleOverrides: {
+        root: {
+          "&.navIcon": {
+            filter: "drop-shadow(0 0 7px #323232)",
+            margin: 18,
+          },
+          "&.navIconSelected": {
+            filter: "drop-shadow(1px 10px 7px #737373)",
+            height: "52px",
+            width: "53px",
+          },
```

Comment:
Are these targeting icons relating to navigation? Wouldn't it be better if we separated it into `navIcon` and `selected` as opposed to grouping into `.navIcon` and `.navIconSelected`?

So selectors would be `&.navIcon.selected`. Are we able to nest rules here like we would in SCSS?

Reply:
I'd have to look into the nested rule feature, but the reason I separated the styles out like this was because the navIcon styles don't apply to the selected styles. The selected Icons are entirely new svgs with slightly bigger dimensions, a different drop shadow, and with their own padding for some reason, so I separated those styles to their own class.

I can definitely change "navIconSelected" to just "selected" though, I just name it that way for readability.

Reply:
Gonna close this out cause I removed these classes entirely in the new commit